aries-site/src/data/wcag/components.json

```diff
@@ -861,6 +861,518 @@
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "conditional",
+        "notes": "An aria attribute can be added."
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.1",
+        "label": "Use of Color",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.2",
+        "label": "Audio Control",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.3",
+        "label": "Contrast (Minimum)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.4",
+        "label": "Resize Text",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.5",
+        "label": "Images of Text",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.6",
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": "Placeholder text fails, other states pass."
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "passed",
```

Comment:
I think this rule may be failing because on the readonly variant of TextInput, when I hover over the tooltip on the readonly button I can't move my pointer onto the tooltip before it disappears.

Reply:
Thanks for the note, I'll update.